### PR TITLE
Clients & Employees Profile Pages

### PR DESCRIPTION
**Pull request by isabellagarrido**
This PR adds the Clients and Employees Profile pages and modals. It also begins to implement the flagged system.

## Client Profile

image

## Employee Profile

image

## Profile Page UI
image

### CODE REVIEW

File: backend/src/user/user.controller.ts [hunk 1]
```diff
@@ -1,4 +1,11 @@
-import { Controller, Get, Param, Post, Request } from '@nestjs/common';
+import {
+  Controller,
+  Get,
+  Param,
+  ParseIntPipe,
+  Post,
+  Request,
+} from '@nestjs/common';
 import { StaffOnly } from 'src/auth/decorators/isStaff.decorator';
```
Comment (Kenneth-Aponte) [inline backend/src/user/user.controller.ts]: this is the import that is causing the build issue, should be relative :)

File: backend/src/user/user.controller.ts [hunk 2]
```diff
@@ -40,7 +56,9 @@ export class UserController {
 
   @StaffOnly()
   @Post('employee/:userId/create')
-  async createEmployee(@Param('userId') userId: number): Promise<UserProfile> {
+  async createEmployee(
```
Comment (Kenneth-Aponte) [inline backend/src/user/user.controller.ts]: maybe rename the same way as the hooks in the frontend.

**Review by Kenneth-Aponte — APPROVED** [review]

**Comment by Kenneth-Aponte** [comment]
Love the header colors in the new profile pages.

File: backend/src/user/user.controller.ts (hunk 2 above)
Comment (isabellagarrido) [reply backend/src/user/user.controller.ts]: fixed in latest commit

File: backend/src/user/user.controller.ts (hunk 1 above)
Comment (isabellagarrido) [reply backend/src/user/user.controller.ts]: ohh, fixed in latest commit

**Pull request merged**